backend/de.metas.business/src/main/java/de/metas/pricing/exceptions/PriceListNotFoundException.java

```diff
@@ -43,14 +41,17 @@
 	private static final AdMessageKey MSG_NO_PO_PRICELIST_FOUND = AdMessageKey.of("NoPOPriceListFound");
 	private static final AdMessageKey MSG_NO_SO_PRICELIST_FOUND = AdMessageKey.of("NoSOPriceListFound");
 
+	private static final AdMessageKey MSG_NO_PO_PRICELIST_FOR_COUNTRIES_FOUND = AdMessageKey.of("NoPOPriceListForCountries");
+	private static final AdMessageKey MSG_NO_SO_PRICELIST_FOR_COUNTRIESFOUND = AdMessageKey.of("NoSOPriceListForCountries");
+
 	public PriceListNotFoundException(final String pricingSystemName, final SOTrx soTrx)
 	{
 		super(getADMessage(soTrx), new Object[] { pricingSystemName });
 	}
 
-	public PriceListNotFoundException(final String pricingSystemName, final SOTrx soTrx, final ArrayList<CountryId> countriesWithoutPrices)
+	public PriceListNotFoundException(final String pricingSystemName, final SOTrx soTrx, final List<CountryId> countriesWithoutPrices)
```

Comment:
when the message is displayed to user make sure the country names are shown and not country IDs